src/components/roverSelector/RoverSelectorPage.tsx

```diff
@@ -1,9 +1,24 @@
 import React from "react";
+import "./RoverSelectorPage.scss";
+import Opportunity from "/images/OpportunityRover.jpg";
+import Spirit from "/images/SpiritRover.jpg";
+import Curiosity from "/images/CuriosityRover.jpg";
 
 export const RoverSelectorPage: React.FunctionComponent = () => {
   return (
-    <main>
-      hello
-    </main>
+    <div className="rover-selector">
+      <div className="rover curiosity">
+        <img src={Curiosity} alt="Opportunity Rover" />
```

Comment:
You copied and pasted these, didn't you? 😉 
```suggestion
        <img src={Curiosity} alt="Curiosity Rover" />
```